app/components/pages/UserProfile.jsx

```diff
@@ -84,26 +113,21 @@ export default class UserProfile extends React.Component {
         }
 
         let followerCount, followingCount;
-        const followers = this.props.global.getIn( ['follow', 'get_followers', accountname] );
-        const following = this.props.global.getIn( ['follow', 'get_following', accountname] );
-
+        const followers = follow ? follow.getIn( ['get_followers', accountname] ) : null;
+        const following = follow ? follow.getIn( ['get_following', accountname] ) : null;
         if(followers && followers.has('result') && followers.has('blog')) {
             const status_followers = followers.get('blog')
             const followers_loaded = status_followers.get('loading') === false && status_followers.get('error') == null
             if (followers_loaded) {
-                followerCount = followers.get('result').filter(a => {
-                    return a.get(0) === "blog";
-                }).size;
+                followerCount = followers.get('count');
```

Comment:
I just moved the counting logic from the component to the reducer, so it's still the old api.